modules/uipath/src/backend/index.ts

```diff
@@ -0,0 +1,105 @@
+import * as sdk from 'botpress/sdk'
+import jsonwebtoken from 'jsonwebtoken'
+
+export class ResponseError extends Error {
+  errorCode: string
+  statusCode: number
+
+  skipLogging = false
+
+  constructor(message: string, statusCode: number, errorCode: string) {
+    super(message)
+    Error.captureStackTrace(this, this.constructor)
+    this.statusCode = statusCode
+    this.errorCode = errorCode
+  }
+}
+
+class UnauthorizedError extends ResponseError {
+  type = 'UnauthorizedError'
+
+  constructor(message: string) {
+    super(`Unauthorized: ${message}`, 401, 'BP_0041')
+  }
+}
+
+const onServerStarted = async (bp: typeof sdk) => {
+  const authRouter = bp.http.createRouterForBot('uipath/auth', {
+    checkAuthentication: true
+  })
+
+  const rootRouter = bp.http.createRouterForBot('uipath', {
+    checkAuthentication: false
+  })
```

Comment:
We try to keep the main `index.ts` as small as possible to make maintaining easier, I suggest moving this in a file called "api.ts". We use the same pattern in all other modules, it makes it easy to find api endpoints